.github/workflows/deployment.yml

```diff
@@ -26,15 +26,15 @@ jobs:
            archivePortable: tar -czf build/distribution/JabRef-portable_macos.tar.gz -C build/distribution JabRef.app && rm -R build/distribution/JabRef.app
 
     runs-on: ${{ matrix.os }}
-    name: Deploy on ${{ matrix.displayName }}
+    name: deploy on ${{ matrix.displayName }}
```

Comment:
The convention seems to be that names are capitalzied (at least if you look at the default built-in jobs that github performs automatically)